ambry-clustermap/src/test/java/com.github.ambry.clustermap/TestUtils.java

```diff
@@ -159,7 +164,7 @@ public static JSONArray getJsonArrayDataNodesRackAware(int dataNodeCount, String
       throws JSONException {
     JSONArray jsonArray = new JSONArray();
     for (int i = 0; i < dataNodeCount; ++i) {
-      jsonArray.put(getJsonDataNode(hostname, basePort + i, sslPort + i, i, hardwareState, disks));
+      jsonArray.put(getJsonDataNode(hostname, basePort + i, sslPort + i, i % 3, hardwareState, disks));
```

Comment:
This function is used to generate test configurations.  I will add an argument to allow the user to set the number of racks